commands/bot/help.js

```diff
@@ -0,0 +1,42 @@
+const { MessageEmbed } = require("discord.js")
+
+module.exports = {
+    name: "help",
+    run: async (message, _, _) => {
+        let embed = new MessageEmbed()
+            .setTitle("What do I got?")
+            .setColor(0x7419b4)
+            .addDescription(
```

Comment:
setDescription() not addDescription()

Reply:
ah ok thats why needed it to test before